modules/utxo-lib/src/bitgo/wallet/Psbt.ts

```diff
@@ -134,3 +158,148 @@ export function signWalletPsbt(
     psbt.signInputHD(inputIndex, signer);
   }
 }
+
+function classifyScriptType(input: PsbtInput): ScriptType2Of3 | undefined {
+  let scriptType: ScriptType2Of3 | undefined;
+  if (Buffer.isBuffer(input.redeemScript) && Buffer.isBuffer(input.witnessScript)) {
+    scriptType = 'p2shP2wsh';
+  } else if (Buffer.isBuffer(input.redeemScript)) {
+    scriptType = 'p2sh';
+  } else if (Buffer.isBuffer(input.witnessScript)) {
+    scriptType = 'p2wsh';
+  }
+  if (Array.isArray(input.tapLeafScript) && input.tapLeafScript.length > 0) {
+    if (scriptType) {
+      throw new Error(`Found both ${scriptType} and p2tr PSBT metadata.`); // can be tested
+    }
+    if (input.tapLeafScript.length > 1) {
+      throw new Error(`Bitgo only supports a single tap leaf script per input.`); // can be tested
```

Comment:
Super Nit: Although this works but prefer to use template literals for string interpolation. Single quotes are preferred for single-line hard-coded strings.

Reply:
Done